exonum/src/helpers/mod.rs

```diff
@@ -16,11 +16,12 @@
 
 use std::env;
 use std::io::{self, Write};
-use std::time::{SystemTime, UNIX_EPOCH};
+use std::time::SystemTime;
 
 use log::{Level, Record, SetLoggerError};
 use env_logger::{Builder, Formatter};
 use colored::*;
+use chrono::prelude::*;
```

Comment:
As far as I can see, you are using only two types (`DateTime` and `Local`). I propose to replace `use chrono::prelude::*` with `use chrono::{DateTime, Local}`.